charts/aws-ebs-csi-driver/templates/servicemonitor.yaml

```diff
@@ -38,6 +38,10 @@ spec:
     - targetPort: 3301
       path: /metrics
       interval: {{ .Values.controller.serviceMonitor.interval | default "15s"}}
+      {{- if .Values.controller.serviceMonitor.metricRelabelings }}
```

Comment:
Why rename this file? It contains other metrics-related stuff not just the ServiceMonitor, please undo the rename